webapp/lib/controller.dart

```diff
@@ -113,5 +132,12 @@ void command(UIAction action, Data data) {
 
       view.contentView.needsReplyAgeHistogram.updateChart(latestData.needsReplyMessagesByDate);
       break;
+
+    case UIAction.systemEventsDataUpdated:
+      Map<DateTime, int> data = new Map.fromIterable(systemEventsDataList,
```

Comment:
I think this is fine for now, but we're going to want to separate between different system restarts.

Reply:
Okay